ledger/blockdb.go

```diff
@@ -97,7 +97,7 @@ func blockGetHdr(tx *sql.Tx, rnd basics.Round) (hdr bookkeeping.BlockHeader, err
 	return
 }
 
-func blockGetCert(tx *sql.Tx, rnd basics.Round) (blk bookkeeping.Block, cert agreement.Certificate, err error) {
+func blockGetEncodedCert(tx *sql.Tx, rnd basics.Round) (blk []byte, cert []byte, err error) {
```

Comment:
You can have the SQL transaction write directly into the `blk` and `cert` return variables; then you don't need to declare `blkbuf` and `certbuf` separately, or return them.

Reply:
done.